index.html

```diff
@@ -0,0 +1,55 @@
+<!DOCTYPE html>
+<html>
+<head>
+<script src="script.js">
+
+// document.getElementById('square1').onclick = function fun(){
+// 	alert("hello");
+// 	};
+
+</script>
+
+
+
+<link rel='stylesheet' href='style.css'>
+	<title></title>
+</head>
+<body>
+
+
+
+<ul>
+<li>
+<div id = "square1" class="square" onclick="function1()">X</div>
+
+<div id = "square2" class="square" onclick="function2()">X</div>
+
+<div id = "square3" class="square" onclick="function3()">O</div>
+</li>
+
+
+<li>
+<div id = "square4" class="square" onclick="function4()">X</div>
+
+<div id = "square5" class="square" onclick="function5()">O</div>
+
+<div id = "square6" class="square" onclick="function6()">X</div>
+</li>
+
+
+
+<li>
+<div id = "square7" class="square" onclick="function7()">O</div>
+
+<div id = "square8" class="square" onclick="function8()">X</div>
+
+<div id = "square9"class="square" onclick="function9()">X</div>
+</li>
+
```

Comment:
I like that you made it work. But I want to say its not usually best practice to have `onclick` attributes inside of your html